src/functionalTest/java/uk/gov/hmcts/reform/waworkflowapi/controllers/CreateTaskTest.java

```diff
@@ -22,23 +28,57 @@
 import static uk.gov.hmcts.reform.waworkflowapi.api.CreateTaskRequestCreator.requestRespondentEvidenceTaskRequest;
 import static uk.gov.hmcts.reform.waworkflowapi.api.CreateTaskRequestCreator.unmappedCreateTaskRequest;
 import static uk.gov.hmcts.reform.waworkflowapi.api.TransitionBuilder.aTransition;
+import static uk.gov.hmcts.reform.waworkflowapi.config.ServiceTokenGeneratorConfiguration.SERVICE_AUTHORIZATION;
 
-@RunWith(SpringIntegrationSerenityRunner.class)
-public class CreateTaskTest {
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@ActiveProfiles("functional")
```

Comment:
It seems we don't have any functional profile or configuration in the project. Therefore, it seems unnecessary.